src/main/java/com/achdev/onlinebookstoreapp/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package com.achdev.onlinebookstoreapp.controller;
+
+import com.achdev.onlinebookstoreapp.dto.BookDto;
+import com.achdev.onlinebookstoreapp.dto.CreateBookRequestDto;
+import com.achdev.onlinebookstoreapp.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties

```suggestion
@RequestMapping("/books")
```

Reply:
Ok